docker-compose.client.rofs.commercial.yml

```diff
@@ -0,0 +1,8 @@
+version: '2.1'
+services:
+
+    #
+    # mender-client
+    #
+    mender-client:
+        image: registry.mender.io/mendersoftware/mender-qemu-rofs-commercial:mender-master
```

Comment:
It should now be removed from https://github.com/mendersoftware/integration/blob/39941fe9712c64011362aae2e86dc241d5b6c59d/other-components-docker.yml#L9-L10